clone.py

```diff
@@ -169,6 +175,17 @@ def _make_filename(self, url):
                         carved_url = yarl.URL(carved_url)
                         if not carved_url.is_absolute():
                             carved_url = self.root.join(carved_url)
+
+                        if carved_url not in self.visited_urls:
+                            yield from self.new_urls.put(carved_url)
+
+    @asyncio.coroutine
+    def run(self):
+        yield from self.new_urls.put(self.root)
+        # Force 404 Page
+        yield from self.new_urls.put(self.error_page)
+        return (yield from self.get_body())
+
                         if carved_url.human_repr() not in self.visited_urls:
```

Comment:
After the release , the whole code of this pr has messed up , please try to run this code and fix the errors :)